data/product.d/fedora.conf

```diff
@@ -6,6 +6,9 @@ product_name = Fedora
 [Network]
 default_on_boot = FIRST_WIRED_WITH_LINK
 
+[Payload]
+enable_addon_repos = False
```

Comment:
Please, set this value in `anaconda.conf` to `True` and drop the changes in `fedora.conf` and `rhel.conf`. We would like to keep these repositories enabled in Fedora.

Reply:
I'll make this change shortly.